packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -179,9 +180,12 @@ export class CalloutContentBase extends BaseComponent<ICalloutProps, ICalloutSta
       calloutMaxWidth
     });
 
-    const overflowStyle: React.CSSProperties = overflowYHidden
-      ? { overflowY: 'hidden', maxHeight: contentMaxHeight }
-      : { maxHeight: contentMaxHeight };
+    const overflowStyle: React.CSSProperties = {
+      ...style,
+      maxHeight: contentMaxHeight,
+      ...(overflowYHidden && { overflowY: 'hidden' })
```

Comment:
This will override any overflowYHidden in the style that's passed in, is that intentional?

Reply:
Maybe for now you can add a comment to the style that's being passed in so users will know that their overflow will get overridden.

Reply:
@pablonete I'm unclear why the style needs to be inline to work?  Why does the conditional style in the `calloutMain` style set not work?

Reply:
The line this discussion hangs out belongs to this change:
> I've had to add the style prop to Callout because there was no continuity in the composition Tooltip - Callout - Popup.

So the point is that inline `style` coming from Tooltip was cut in Callout, there was no way to pass it down.

Then I'm using the `overflowY` in the style as a flag, because `needsVerticalScrollBar` is also modifying the final inline style but giving precedence to the user provided style.